hydrolib/core/dflowfm/tim/serializer.py

```diff
@@ -65,7 +68,9 @@ def _serialize_timeseries_lines(data, config) -> List[str]:
     @staticmethod
     def _serialize_to_timeseries_block(data, format_float) -> TimeSeriesBlock:
```

Comment:
The typehinting is missing for a couple of the functions in this class!

Reply:
do you know what type format_float is?

Reply:
`Callable[[float], str]`